sources/web/datalab/polymer/components/datalab-files/datalab-files.html

```diff
@@ -174,10 +174,13 @@
         <iron-icon icon="add-box"></iron-icon>
         <span>Folder</span>
       </paper-button>
-      <paper-button class="toolbar-button">
+      <paper-button class="toolbar-button" on-click="_altUpload">
```

Comment:
Can you post a screenshot? I don't see how the user selects a file to upload if the file input field is hidden.